src/wrappers/SpecimenWrapper.js

```diff
@@ -83,56 +82,61 @@ class SpecimenWrapper {
     return specimen;
   }
 
+  /**
+   * Get the catalogNumber if present.
+   */
   get catalogNumber() {
     // Get the catalog number from the specimen object if present.
     if (has(this.specimen, 'catalogNumber')) return this.specimen.catalogNumber;
 
     // Otherwise, try to parse the occurrenceID and see if we can extract a
     // catalogNumber from there.
     if (has(this.specimen, 'occurrenceID')) {
-      const specimen = SpecimenWrapper.createFromOccurrenceID(this.specimen.occurrenceID);
+      const specimen = SpecimenWrapper.fromOccurrenceID(this.specimen.occurrenceID);
       if (has(specimen, 'catalogNumber')) return specimen.catalogNumber;
     }
     return undefined;
   }
 
+  /**
+   * Get the institutionCode if present.
+   */
   get institutionCode() {
     // Get the institution code from the specimen object if present.
     if (has(this.specimen, 'institutionCode')) return this.specimen.institutionCode;
 
     // Otherwise, try to parse the occurrenceID and see if we can extract an
     // occurrenceID from there.
     if (has(this.specimen, 'occurrenceID')) {
-      const specimen = SpecimenWrapper.createFromOccurrenceID(this.specimen.occurrenceID);
+      const specimen = SpecimenWrapper.fromOccurrenceID(this.specimen.occurrenceID);
       if (has(specimen, 'institutionCode')) return specimen.institutionCode;
     }
     return undefined;
   }
 
+  /**
+   * Get the collectionCode if present.
+   */
   get collectionCode() {
     // Get the collection code from the specimen object if present.
     if (has(this.specimen, 'collectionCode')) return this.specimen.collectionCode;
 
     // Otherwise, try to parse the occurrenceID and see if we can extract an
     // occurrenceID from there.
     if (has(this.specimen, 'occurrenceID')) {
-      const specimen = SpecimenWrapper.createFromOccurrenceID(this.specimen.occurrenceID);
+      const specimen = SpecimenWrapper.fromOccurrenceID(this.specimen.occurrenceID);
       if (has(specimen, 'collectionCode')) return specimen.collectionCode;
     }
     return undefined;
   }
 
+  /**
+   * Return the occurrence ID of this specimen, if we have one. Otherwise, we
+   * attempt to construct one in the form:
+   *  "urn:catalog:" + institutionCode (if present) + ':' +
+   *  collectionCode (if present) + ':' + catalogNumber (if present)
+   */
```

Comment:
We previously recorded specimen identifiers as Darwin Core Triplets at three separate fields (institutionID, collectionID, catalogNumber), but I figured in this PR that it made sense to combine them into a single field so that they could be edited more easily from the Curation Tool. I was pretty sure based on its documentation that [`dwc:occurrenceID`](https://terms.tdwg.org/wiki/dwc:occurrenceID) was the right way of do this. I've also just recorded some examples of other data providers using this model in https://github.com/phyloref/clade-ontology/issues/61#issuecomment-500627446.

I should definitely use [`dwc:basisOfRecord`](https://terms.tdwg.org/wiki/dwc:basisOfRecord) to make it clear that this is a specimen, not just any occurrence, but apart from that I'm not sure what other field we could use to store the specimen ID here. Am I forgetting some property that I should use instead?

Reply:
I've just added `dwc:basisOfRecord` entries to specimens in this PR so they're no longer pure occurrence IDs. Should I merge this PR while we discuss the basis way of representing specimens in https://github.com/phyloref/clade-ontology/issues/61#issuecomment-500627446? Otherwise, I could take out specimens entirely from this PR and then we could put them back in once we have a better plan to implement them.